stackgl_modules/index.js

```diff
@@ -1,43 +1,33 @@
-(function webpackUniversalModuleDefinition(root, factory) {
-	if(typeof exports === 'object' && typeof module === 'object')
-		module.exports = factory();
-	else if(typeof define === 'function' && define.amd)
-		define([], factory);
-	else if(typeof exports === 'object')
-		exports["Stackgl"] = factory();
-	else
-		root["Stackgl"] = factory();
-})(self, function() {
-return /******/ (function() { // webpackBootstrap
+/******/ (function() { // webpackBootstrap
 /******/ 	var __webpack_modules__ = ({
 
-/***/ 7386:
+/***/ 1964:
```

Comment:
Please note these index changes are coming from #6665 since between v2.24.2 and v2.24.3.
And there was no need to rebuild the stackgl index since 2.19.0 when `is-mobile` is updated in #6517.